pages/wg/dei.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: Diversity, Equity, & Inclusion Working Group
+subtitle:
+permalink: /wg/dei/
+---
+
+# Overview
+
+The US-RSE Association is committed to providing an inclusive environment with equitable treatment for all and to promoting and encouraging diversity throughout the RSE community in the US. The US-RSE Diversity, Equity, and Inclusion (DEI) working group is for those who are interested in regular discussions regarding DEI in the RSE community. The goal of the DEI working group is to promote the values of the US-RSE DEI Mission Statement with a particular focus on promoting opportunities for DEI education to the wider community. The group meets bi-weekly via Zoom and welcomes all newcomers.
```

Comment:
Is there any reason to scope to "in the US"? It's subtly implied since we are US-RSE, but this could also be read as "Oh, if we're at an international conference we will turn a head to these issues because we only care about the US." Obviously this isn't the case, but I think it will be more succinct and not make people think this "only the US?" if we just remove "in the US."

```suggestion
The US-RSE Association is committed to providing an inclusive environment with equitable treatment for all and to promoting and encouraging diversity throughout the RSE community. The US-RSE Diversity, Equity, and Inclusion (DEI) working group is for those who are interested in regular discussions regarding DEI in the RSE community. The goal of the DEI working group is to promote the values of the US-RSE DEI Mission Statement with a particular focus on promoting opportunities for DEI education to the wider community. The group meets bi-weekly via Zoom and welcomes all newcomers.
```

Reply:
That is a copy-pasta from the first line of the US-RSE DEI Mission statement: https://us-rse.org/about/dei-mission/

I can't remember why we decided to include that language specifically in the mission statement... But there was a conversation about it.

Reply:
oh gotcha, then it would be okay to leave as is. I wasn't part of the original discussion, but my initial reaction was reading "in the US" and thinking "huh, that sounds funny... why would we explicitly create this geographic boundary?" I don't see why our DEI-WG would not want to include people outside the US, or at least explicitly not say something that could imply that. But if it was already discussed, probably okay to leave as is.

Reply:
> The US-RSE Diversity, Equity, and Inclusion (DEI) working group is for those who are interested in regular discussions regarding DEI in the RSE community.

... reads slightly oddly to me, I'd have thought it's more for those who want to work together in furthering the group's goals but, given the emphasis on regularity, maybe the point is to emphasize that you need members to be able to turn up to all the media club events or something?

Reply:
I wonder if we should just leave this as it is on the DEI mission statement for now, and then discuss a change in the DEI group later.

Reply:
I'm fine with that. We can also leave this PR open till Monday and discuss it then.

Reply:
Either works for me.

Reply:
either is also ok with me